<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>700</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Смайлик Пейнтер</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QPushButton" name="colorButton">
                                <property name="text">
                                    <string>Выбрать цвет смайлика</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="colorLabel">
                                <property name="minimumSize">
                                    <size>
                                        <width>100</width>
                                        <height>30</height>
                                    </size>
                                </property>
                                <property name="frameShape">
                                    <enum>QFrame::Panel</enum>
                                </property>
                                <property name="frameShadow">
                                    <enum>QFrame::Sunken</enum>
                                </property>
                                <property name="text">
                                    <string>RGB(255,255,0)</string>
                                </property>
                                <property name="alignment">
                                    <set>Qt::AlignCenter</set>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="QGroupBox" name="groupBox">
                        <property name="title">
                            <string>Масштаб смайлика</string>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_2">
                                    <item>
                                        <widget class="QLabel" name="label">
                                            <property name="text">
                                                <string>Текущий масштаб:</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="scaleLabel">
                                            <property name="text">
                                                <string>100%</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <widget class="QSlider" name="scaleSlider">
                                    <property name="minimum">
                                        <number>25</number>
                                    </property>
                                    <property name="maximum">
                                        <number>200</number>
                                    </property>
                                    <property name="value">
                                        <number>100</number>
                                    </property>
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout_3">
                                    <item>
                                        <widget class="QLabel" name="label_2">
                                            <property name="text">
                                                <string>25%</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <spacer name="horizontalSpacer_2">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>40</width>
                                                    <height>20</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="label_3">
                                            <property name="text">
                                                <string>200%</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item>
                    <widget class="QWidget" name="smileyWidget">
                        <property name="minimumSize">
                            <size>
                                <width>400</width>
                                <height>400</height>
                            </size>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>